assets/js/modules/idea-hub/datastore/draft-post-ideas.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * `modules/idea-hub` data store: draft-post-ideas.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from './constants';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+
+const fetchGetDraftPostIdeasStore = createFetchStore( {
+	baseName: 'getDraftPostIdeas',
+	controlCallback: () => {
+		return API.get( 'modules', 'idea-hub', 'draft-post-ideas' );
+	},
+	reducerCallback: ( state, draftPostIdeas ) => {
+		return {
+			...state,
+			draftPostIdeas,
+		};
+	},
+} );
+
+const baseInitialState = {
+	draftPostIdeas: [],
```

Comment:
The initial value should be `undefined` which indicates that data hasn't been loaded yet.

Reply:
Updated @eugene-manuilov Can you check?